src/components/seo/jobcard/jobcard.module.scss

```
@@ -0,0 +1,88 @@
+@use '@/styles/base/utils' as *;
+
+
+.jobCard {
+  @media (min-width: 768px) {
+    width: 1044px;
+  }
```

Nope ?? never do this. ever 

```suggestion
width: 100%; 
```

does the trick

---

Done